src/main/java/seedu/fridgefriend/command/AddCommand.java

```diff
@@ -56,8 +56,8 @@ public void execute() throws RepetitiveFoodIdentifierException {
 
     private void addFood() throws RepetitiveFoodIdentifierException {
         assert foodToAdd != null : "Unable to add a null food";
-        Storage.saveHistoryData(foodToAdd);
         fridge.add(foodToAdd);
+        Storage.saveHistoryData(foodToAdd);
```

Comment:
Shifting the order of the foodToAdd would result in the history log data adding the post-edited quantity instead of the original quantity. Has this been addressed?